sonar-project.properties

```diff
@@ -1,5 +1,5 @@
-sonar.projectKey=com.telefonica.iot:fiware-orion-pep
-sonar.projectName=fiware-orion-pep
+sonar.projectKey=com.telefonica.iot:fiware-pep-steelskin
+sonar.projectName=fiware-pep-steelskin
```

Comment:
I will talk with the Sonar team in order to change the project name once this PR will be merged.